components/Button/ApplyButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { ChevronIcon } from 'components/Icons/ChevronIcon';
+import styles from './ApplyButton.module.scss';
+
+type ApplyButtonProps = {
+    className?: string;
+    url: string;
+    title: string;
+    children: React.ReactNode;
+};
+
+export const ApplyButton = ({
+    className,
+    url,
+    title,
+    children,
+}: ApplyButtonProps) => (
+    <a
+        href={url}
+        className={[styles.button, styles.radialBackground, className].join(
+            ' ',
+        )}
+    >
+        <div className={styles.overflowHider}>
+            <div className={styles.swishBackground} />
+        </div>
+        <div className={styles.title}>
+            {title}{' '}
+            <ChevronIcon className={styles.chevron} width={28} height={40} />
```

Comment:
Theres an issue with the alignment of the chevron when the size changes. could we remove position absolute and use `display: flex; align-items: center;` on the title class.
![2022-02-28 17 27 10](https://user-images.githubusercontent.com/35934884/156031167-915db51b-68af-40f3-9325-10883bda4dcd.gif)

Reply:
I've gone for `align-items: baseline` for this one